composer.json

```
@@ -90,8 +90,8 @@
         "drupal/redirect": "^1",
         "drupal/responsive_favicons": "^2.0",
         "drupal/scheduler": "^2 || ^2.0@RC",
-        "drupal/search_api": "1.17 || ^1.19",
-        "drupal/search_api_solr": "^4.1",
+        "drupal/search_api": "^1.35",
```

https://www.drupal.org/project/search_api 8.x-1.35 `Works with Drupal: ^10.1 || ^11` but we still allow Drupal Core 10.0.x. It looks like https://www.drupal.org/project/search_api/releases/8.x-1.31 would allow 10.0.x. I _think_ ??? if we use `"^1.31"` it would use the older version with 10.0.x and the newer version with 10.1/2?

```suggestion
        "drupal/search_api": "^1.31",
```